spec.md

```diff
@@ -112,6 +113,43 @@ This header allows brokers to reject requests from marketplaces for versions the
 The marketplace MUST authenticate with the service broker using HTTP
 basic authentication (the `Authorization:` header) on every request. The broker is responsible for validating the username and password and returning a `401 Unauthorized` message if credentials are invalid. It is RECOMMENDED that brokers support secure communication from platform marketplaces over TLS.
 
+## Originating Identity
+
+Often a service broker will need to know the identity of the user that
+initiated the request from the platform. For example, this might be needed
+for auditing or authorization purposes. In order to facilitate this, the
+platform will need to provide this identification information to the broker
+on each request. If the platform wishes to provide this information then
+it MUST include an additional HTTP header on all requests it sends to the
+broker, except the request to retrieve the catalog since that is not
+associated with a platform's user. The format of the header MUST be:
```

Comment:
Two queries:
1. Do you want to audit who registered the service broker? In CF, whoever calls `cf create-service-broker` will implicitly call the catalog endpoint, so we could record whoever ran that command.
2. In some cases (ie. orphan mitigation), their will not be an originating user. In this case I guess the platform will just omit this header.
3. Following the above, is the sentence `If the platform wishes to provide this information then it MUST ...` strange? Maybe it could just read `Platforms wishing to provide this information CAN ...`. Combining a MUST with an optional thing seems odd reflecting back on this.

Reply:
not hearing any objection, I made this change.